app/main/lib/serializer.py

```diff
@@ -0,0 +1,16 @@
+import json
+import datetime
+
+def safe_serializer(obj):
```

Comment:
`date_serializer`?

Reply:
I went back and forth on that name. I chose `safe` because, where it is used, the json serialize option doesn't care about the fact that this is a specific, narrow serialization, and over time, this would be the singular location to introduce safe serializations by type